cmd/snap/cmd_prefer.go

```diff
@@ -0,0 +1,68 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"github.com/snapcore/snapd/i18n"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type cmdPrefer struct {
+	Positionals struct {
+		Snap installedSnapName `required:"yes"`
+	} `positional-args:"true"`
+}
+
+var shortPreferHelp = i18n.G("Prefer a manual alias or an entire snap")
```

Comment:
Can we prefer manual aliases?

Either way the summary feels strange.. perhaps:

Perhaps: `"Prefer aliases from snap and disable conflicts"`